src/utils/feature-flags.ts

```diff
@@ -1,67 +1,80 @@
-import { initialize, LDFlagSet } from 'launchdarkly-js-client-sdk'
+import { initialize, LDClient, LDFlagSet, LDOptions, LDUser } from 'launchdarkly-js-client-sdk'
```

Comment:
This is a rewrite of this file that I did a while back in Edit UI.
The main changes from before are:
- better init (with configured options)
- use shared user key 'anonymous' for unknown users (no more random keys)
- updated way to get FF, which supports live flag updates
- a way to set (identify) user, to support per-user flags

(Yes, this file should be updated in the other projects - see /bcgov/entity#6093.)